src/gen_rpc_client.erl

```diff
@@ -66,27 +74,27 @@ stop(NodeOrTuple) when ?is_node_or_tuple(NodeOrTuple) ->
 %%% Server functions
 %%% ===================================================
 %% Simple server call with no args and default timeout values
--spec call(node_or_tuple(), atom() | tuple(), atom() | function()) -> term() | {badrpc,term()} | {badtcp,term()}.
+-spec call(node_or_tuple(), atom() | tuple(), atom()) -> term() | {badrpc,term()} | {badtcp,term()}.
 call(NodeOrTuple, M, F) ->
     call(NodeOrTuple, M, F, [], undefined, undefined).
 
 %% Simple server call with args and default timeout values
--spec call(node_or_tuple(), atom() | tuple(), atom() | function(), list()) -> term() | {badrpc,term()} | {badtcp,term()}.
+-spec call(node_or_tuple(), atom() | tuple(), atom(), list()) -> term() | {badrpc,term()} | {badtcp,term()}.
 call(NodeOrTuple, M, F, A) ->
     call(NodeOrTuple, M, F, A, undefined, undefined).
 
 %% Simple server call with custom receive timeout value
--spec call(node_or_tuple(), atom() | tuple(), atom() | function(), list(), timeout()) -> term() | {badrpc,term()} | {badtcp,term()}.
+-spec call(node_or_tuple(), atom() | tuple(), atom(), list(), timeout() | undefined) -> term() | {badrpc,term()} | {badtcp,term()}.
 call(NodeOrTuple, M, F, A, RecvTO) ->
     call(NodeOrTuple, M, F, A, RecvTO, undefined).
 
 %% Simple server call with custom receive and send timeout values
 %% This is the function that all of the above call
--spec call(node_or_tuple(), atom() | tuple(), atom() | function(), list(), timeout() | undefined, timeout() | undefined) ->
+-spec call(node_or_tuple(), atom() | tuple(), atom(), list(), timeout() | undefined, timeout() | undefined) ->
     term() | {badrpc,term()} | {badtcp,term()}.
 call(NodeOrTuple, M, F, A, RecvTO, SendTO) when ?is_node_or_tuple(NodeOrTuple), is_atom(M) orelse is_tuple(M), is_atom(F), is_list(A),
-                                         RecvTO =:= undefined orelse ?is_timeout(RecvTO),
```

Comment:
Let me know if you prefer `is_timeout` (as was).

Reply:
Nah, whatever there was in me that cared about the finer points of macro-presented-like-functions has long dissolved into benevolent indifference.

Reply:
(But thanks for asking)